lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -1,6 +1,11 @@
 <domain type='<%= domain_type %>'>
   <name><%= name %></name>
   <memory><%= memory_size %></memory>
+<% if !hugepages.empty? -%>
```

Comment:
I think that should rather be a boolean and then handled like `if hugepages`

Reply:
I agree with @mmoll. Can you change this?